composer.json

```diff
@@ -20,8 +20,7 @@
         "cakephp/cakephp": "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.1.*",
-        "cakephp/cakephp-codesniffer": "dev-master"
```

Comment:
To avoid multiple per project copies of codesniffer locally. Locally most have codesniffer installed globally and for travis it's installed for the phpcs build. We follow this in almost all core and foc repos.